index.html

```diff
@@ -0,0 +1,292 @@
+              <a href=""> <img src="images/twitter.png" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Linkedin.png" alt="LinkedIn" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/medium.png" alt="Media" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Vector-1.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Vector-3.png" alt="image" /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="wrapper2">
+        
+        
+
+        <div id="portfolio" class="works"></div>
+
+        <!-- desktop popup -->
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+           <div class="desktopIntro"> <h2 class="deskHeading"></h2>
+            <ul class="deskButtons">
+              <li class="deskList1"  ></li>
+              <li class="deskList2"></li>
+              <li  class="deskList3"></li>
+            </ul>
+            <div class="closeProj"><i class="fa fa-times" onclick="closeall()"></i></div>
+           </div>
+            <div><img src="/images/snap3.png" class="deskImag" alt="project-image"></div>          
+            <div class="desktopFlex">
+            <p class="deskpara"></p>
+            <aside>
+            <ul class="deskButtonlist">
+                           
+            </ul>
+            <div class="liner"></div>
+            <div class="separater">
+            <div class="seeLive">
+           <a href="" id="liveLink"> see Live  <img src="images/live.png" alt="github pages"> </a></div>
+           <div class="seeSource">
+            <a href="" id="sourceLink"> See Source <img src="images/vect1.png" alt="github"> </a></div></div>
+          </aside></div>
+          </section>
+        </div>
+
+
+
+  <!-- mobile popup -->
+  <div id="mobileWrap" class="modal">
+    <section class="mobileDetail">
+     <div class="mobileIntro"> <h2 class="mobileHeading"></h2>
+      <ul class="mobileButtons">
+        <li class="mobList1"  ></li>
+        <li class="mobList2"></li>
+        <li  class="mobList3"></li>
+        
+      </ul>
+      <div class="closeProj"><i class="fa fa-times" onclick="closeall()"></i></div>
+     </div>
+      <div><img src="images/" class="mobImag" alt="project-image"></div>          
+      <div class="mobileFlex">
+      <p class="mobilepara"></p>
+      <aside>
+      <ul class="mobButtonlist">
+     
+                    
+      </ul>
+      <div class="linerMobile"></div>
+      <div class="separaterMob">
+      <div class="seeLiveMob">
+     <a href="" id="liveLinkM"> see Live <img src="images/live.png" alt="github pages"> </a></div>
+     <div class="seeSource">
+      <a href="" id="sourceLinkM"> See Source <img src="images/vect1.png" alt="github"> </a></div></div>
+    </aside></div>
+    </section>
+  </div>
+  <!-- mobile popup -->
```

Comment:
- [ ] I noticed that your popup code is repetitive I would like to recommend refactoring your code and making it **DRY**  (Don't Repeat Yourself) [follow this article](https://www.taniarascia.com/refactoring-dont-repeat-yourself/)  and just make it responsive for both mobile and desktop :smile: